RecoPixelVertexing/PixelTriplets/python/quadrupletseedmerging_cff.py

```diff
@@ -48,3 +60,4 @@
 )
 eras.trackingPhase1.toModify(PixelSeedMergerQuadruplets, **_forPhase1)
 eras.trackingPhase1PU70.toModify(PixelSeedMergerQuadruplets, **_forPhase1)
+eras.trackingPhase2PU140.toModify(PixelSeedMergerQuadruplets, layerList = _layerListForPhase2)
```

Comment:
Probably I will need it, but in this moment I am not using the Presplitting anywhere..